cdisutils/storage3.py

```diff
@@ -110,6 +110,7 @@ def load_creds():
     }
     s3_endpoint_defaults = {
         "ceph": "ceph.service.consul",
+        "cephb" : "gdc-cephb-objstore.osdc.io",
```

Comment:
Why do we need to add this again considering ceph has been decommissioned? Are there files with that s3_type?

Reply:
We don't. The feature branch and develop are out of sync and these changes were on develop. This PR is partially a sync PR. The tests are failing on the branch I'm merging into so I fixed them as well.